app/gui/view/component-browser/component-group/src/lib.rs

```diff
@@ -29,14 +29,15 @@ use ensogl_gui_component::component;
 use ensogl_hardcoded_theme::application::component_browser::component_group as theme;
 use ensogl_list_view as list_view;
 use ensogl_text as text;
+use ensogl_text_embedded_fonts_names as embedded_fonts_names;
 
 
 
 // =================
 // === Constants ===
 // =================
 
-const HEADER_FONT: &str = "DejaVuSans-Bold";
+const HEADER_FONT: &str = DEJA_VU_SANS_BOLD;
```

Comment:
use something like `fonts::DEFAULT` instead, so we will not hardcode this font globally. Also, fonts should be really read from the theme config, why not move it to theme instead?

Reply:
In such a case, please close this PR or switch it to draft until it's ready.

Reply:
> In such a case, please close this PR or switch it to draft until it's ready.

Uhh, sorry for a miscommunication on my side: this comment is already addressed in the PR, that's why the PR is now non-Draft!